cos_alerter/alerter.py

```diff
@@ -30,15 +33,29 @@ def set_path(self, path: str):
         """Set the config file path."""
         self.path = Path(path)
 
+    def _hash_keys(self, clients):
+        """Hash the keys in the clients dictionary."""
+        for client_info in clients.values():
+            client_key = client_info.get("key", "")
+            if client_key:
+                hashed_key = hashlib.sha256(client_key.encode()).hexdigest()
+                client_info["key"] = hashed_key
+
```

Comment:
I think I may not have been clear about the keys. The reason we want to hash incoming keys is to make sure we are not storing passwords on disk. If we store the keys in the config file, there is no point in the hashing. The values in the config file should already be hashed.